packages/react/scripts/build-components.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const path = require('path');
+const fs = require('fs-extra');
+const {execSync} = require('child_process');
+const packageJson = require('../package.json');
+const fixFiles = require('./fix-files');
+const {logger} = require('@oxygen-ui/logger');
+
+const outputDir = path.resolve(__dirname, '..', 'dist', 'transpiled');
+
+logger.log('💅 Transpiling components...');
+execSync(
+  `npx tsc --project tsconfig.lib.json --declarationDir ${outputDir} --emitDeclarationOnly false --outDir ${outputDir}`,
+);
+logger.log('💅 Components transpiled! 🎉🎉🎉');
+
+logger.log('💅 Copying components...');
+fs.rmSync(path.resolve(__dirname, '..', 'dist/transpiled/src/components/index.js'));
```

Comment:
Shall we use ` path.join`?
```suggestion
fs.rmSync(path.resolve(__dirname, '..', path.join('dist', 'transpiled', 'src', 'components', 'index.js'));
```